vizro-core/docs/pages/user-guides/selectors.md

```diff
@@ -143,4 +143,4 @@ indicating that [pandas.api.types.is_datetime64_any_dtype()](https://pandas.pyda
     [Filter]: ../../assets/user_guides/selectors/default_filter_selectors.png
 
 
-To enhance existing selectors, please see our How-to-guide on creating [custom components](custom-components.md).
+To enhance existing selectors, see our how-to-guide on creating [custom components](custom-components.md).
```

Comment:
I'm trying to find this link Antony once sent to me, but I can't find it quickly ..
But basically it was a link to the best practices of Kedro docs, so you might be familiar with it @stichbury .

It provided some guidance that one should not use "custom components" as a hyperlink but "creating custom components", so always providing context of what the hyperlink contains instead of only having "custom components" be the hyperlink. Not sure if I make any sense 😄

Reply:
Here it is: https://github.com/kedro-org/kedro/wiki/Kedro-documentation-style-guide#links

```
Links
Make hyperlink descriptions as descriptive as you can. This is a good description:

Learn how to [update the project pipeline](https://docs.kedro.org/en/stable/tutorial/create_pipelines.html#update-the-project-pipeline)

This is less helpful:

Learn how to update the [project pipeline](https://docs.kedro.org/en/stable/tutorial/create_pipelines.html#update-the-project-pipeline)
```
Don't write this:

To learn how to update the project pipeline, see [here](https://docs.kedro.org/en/stable/tutorial/create_pipelines.html#update-the-project-pipeline)

Reply:
You're right! This set of fixes is just to get Vale to pass rather than to make all the changes necessary to perfect the docs, so I've not tackled everything like this in this PR (although it sure does look like it from the number of changes!). It would be great to have Vale check hyperlinks for content -- I'm not sure it can do that but I have asked on their discussion forum https://github.com/errata-ai/vale/discussions/807

I have ticketed it (on this repo, it's a nice external issue for a potential contributor) https://github.com/mckinsey/vizro/issues/424

Reply:
Perfect, thanks!